getting_started.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Getting Started with Overwatch
+---
+
+Overwatch is a security platform aimed at end users, it's purpose is to help keep track of your devices and servers preventing external access or detecting irregluar behavior that might otherwise go unnoticed.
+
+
+## Overwatch Devices
+
+Overwatch uses an installable client agent to monitor your devices, analyzing and logging it's regular behavior and enforcing the security rules that you configure on our web application. The term `Device` refers to any platform or physical hardware on which our client agent can be installed.
+
+## The Overwatch Agent
+
+Our agent runs as a service on your device, it's purpose is two-fold: it enforces any [Security Policy rules](#security-policies) that you've defined in the web application, and it sends regular status updates to the Overwatch servers for analysis.
+
+The stats sent out by the agent include plain keepalives, inbound and outbound data traffic, open connections and general details such as the device's IP. The privacy of our user's data is very important to us, as such all communication between the agent and the Overwach server is encrypted, and while we do analyze the behaviour of the device - such as the amount of traffic exchanged between the device and the network, the actual data is never sent down the wire. If you have any concerns about privacy, please [get in touch](get-in-touch) and we'll be happy to discuss our security measures in more detail.
```

Comment:
The `get in touch` link should be `[get in touch][get-in-touch]`. Using parens means that's the explicit URL. Callouts to links elsewhere are done with square brackets.